service/src/test/java/uk/gov/hmcts/reform/fpl/service/robotics/SampleRoboticsTestDataHelper.java

```diff
@@ -123,14 +123,14 @@ public static Set<Child> expectedChildren() {
         return ImmutableSet.of(Child.builder()
                 .firstName("Sarah")
                 .lastName("Reeves")
-                .gender("FEMALE")
+                .gender("Girl")
```

Comment:
Sorry, I am a bit confuse on this one.
If it was male/female for robotics, why do we need to change it to Boy/Girl?